substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/Range.java

```diff
@@ -36,6 +36,10 @@
  */
 
 public class Range {
+
+    /* Use '.' for PECOFF files */
+    private static final String CLASS_DELIMITER = System.getProperty("os.name").toLowerCase().contains("windows") ? "." : "::";
```

Comment:
It would be great if we find a CLASS_DELIMITER that works for both platforms.

Reply:
Well, I only used :: because I was mimicking the format used for C++ code (gdb doesn't really understand Java). Once we embed DWARF method info records into a DWARF type info record this ought not to be a concern. So, I am happy to go with '. as the separator.

Reply:
Yes. Please lets use `.` as CLASS_DELIMITER for all platforms.

Reply:
No, please change it to use '.' for both.